app/routes/resources+/note-editor.tsx

```diff
@@ -89,6 +89,10 @@ export function NoteEditor({
 		onValidate({ formData }) {
 			return parse(formData, { schema: NoteEditorSchema })
 		},
+		defaultValue: {
+			title: note?.title,
+			content: note?.content,
+		},
```

Comment:
Setting the default value here will allows Conform to persists the input value if there is a previous submission.